newIDE/app/src/EventsBasedBehaviorsList/index.js

```diff
@@ -22,9 +22,30 @@ import {
   unserializeFromJSObject,
 } from '../Utils/Serializer';
 import { type UnsavedChanges } from '../MainFrame/UnsavedChangesContext';
+import Tooltip from '@material-ui/core/Tooltip';
+import VisibilityOffIcon from '@material-ui/icons/VisibilityOff';
 
 const EVENTS_BASED_BEHAVIOR_CLIPBOARD_KIND = 'Events Based Behavior';
 
+const renderEventsBehaviorLabel = (
+  eventsBasedBehavior: gdEventsBasedBehavior
+) =>
+  eventsBasedBehavior.isPrivate() ? (
+    <>
+      <Tooltip title="This function won't be visible in the events editor">
+        <VisibilityOffIcon
+          fontSize="small"
+          style={{ marginRight: 5, verticalAlign: 'bottom' }}
+        />
+      </Tooltip>
+      <span title={eventsBasedBehavior.getName()}>
+        {eventsBasedBehavior.getName()}
+      </span>
+    </>
+  ) : (
+    eventsBasedBehavior.getName()
```

Comment:
should this be a variable instead of calling the function 3 times? (not sure it improves performance...)